docs/50_deploy.md

```diff
@@ -36,4 +36,9 @@ spec:
             - sleep
           args:
             - 30s
-```
\ No newline at end of file
+```
+
+## Await resources completion
+
+During the deploy phase resources are applied sequentially one after the other. `mlp` allows to wait for the completion of a resource using the annotation `mia-platform.eu/await-completion` on the specific resource to be awaited. The annotation value should be a duration string as expected by the [`time.ParseDuration`](https://pkg.go.dev/time#ParseDuration) function that specifies the time to wait for completion before a timeout error is triggered. 
```

Comment:
This part it's too much technical. Can you add some duration example? For example, 30s, 1m